.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
as we discussed this is not needed. prettier should **only** be run via eslint, and you shouldn't have any issues...